keyboards/designedbygg/darkmage/keymaps/default/keymap.c

```diff
@@ -52,3 +52,12 @@ const uint16_t PROGMEM encoder_map[][1][2] = {
     [1]   = {ENCODER_CCW_CW(RGB_VAD, RGB_VAI) },
 };
 #endif
+
+#if defined(CAPS_LOCK_KEY)
+bool rgb_matrix_indicators_advanced_user(uint8_t led_min, uint8_t led_max) {
+    if (host_keyboard_led_state().caps_lock) {
+        RGB_MATRIX_INDICATOR_SET_COLOR(29, 255, 255, 255);
```

Comment:
```suggestion
        RGB_MATRIX_INDICATOR_SET_COLOR(CAPS_LOCK_LED_INDEX, RGB_WHITE);
```
no point in declaring the LED but not using it. Also see https://github.com/SonixQMK/qmk_firmware/blob/sn32_develop/quantum/color.h